src/ApiUrlInterpreter.ts

```diff
@@ -187,7 +187,7 @@ export default class ApiUrlInterpreter {
             return;
         }
 
-        const replyMessages = await message.channel.send(`Making a request to ${path!.name}`);
+        const replyMessages = await message.channel.send(`Making a request to \`${path!.name}\``);
```

Comment:
This change seems very unnecessary.

Reply:
Well yes, it was just for formatting:

FROM:
![grafik](https://github.com/Querijn/BottyMcBotface/assets/76645494/d9bc1d27-d51e-4b7e-96ec-ba98b7d5f191)


TO:
![grafik](https://github.com/Querijn/BottyMcBotface/assets/76645494/e4044b68-1185-4c0e-9c60-344863e78a68)

Reply:
I reverted it